fullscreen.bs

```diff
@@ -171,12 +174,17 @@ partial interface Document {
  <dd><p>Returns true if <var>document</var> has the ability to display <a>elements</a> fullscreen
  and <a>fullscreen is supported</a>, or false otherwise.
 
- <dt><code><var>document</var> . {{Document/fullscreenElement}}</code>
- <dd><p>Returns <var>document</var>'s <a>fullscreen element</a>.
-
  <dt><code><var>promise</var> = <var>document</var> . {{Document/exitFullscreen()}}</code>
  <dd><p>Stops <var>document</var>'s <a>fullscreen element</a> from being displayed fullscreen and
  fulfills <var>promise</var> when done.
+
+ <dt><code><var>document</var> . {{DocumentOrShadowRoot/fullscreenElement}}</code>
+ <dd><p>Returns the result of retargeting <var>document</var>'s <a>fullscreen element</a> against
+ <var>document</var>.
```

Comment:
I'm not sure about the use of "retargeting" since we never explain it outside of an algorithm. Perhaps we should just keep this vague and say "document's fullscreen element" and "shadow root's fullscreen element" below?